apps/src/applab/dataBrowser/DataTable.jsx

```diff
@@ -138,6 +138,32 @@ const DataTable = React.createClass({
     return `column${i}`;
   },
 
+  importCsv(csvData) {
+    FirebaseStorage.importCsv(
+      this.props.tableName,
+      csvData,
+      () => this.setState(this.getInitialState()),
+      msg => console.warn(msg));
+  },
+
+  exportCsv() {
+    const tableName = encodeURIComponent(this.props.tableName);
+    location.href = `/v3/export-firebase-tables/${Applab.channelId}/${tableName}`;
+  },
+
+  /** Delete all rows, but preserve the columns. */
+  clearTable() {
+    const msg = 'Do you really want to delete all data from this table? ' +
+      'You cannot undo this action.';
```

Comment:
It sounds like we're not being strict about localizing messages which are specific to applab for now. However I can see some value in extracting longer UI messages, so I've localized the two that you've commented on.